elasticsearch/pom.xml

```diff
@@ -54,6 +54,31 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
> I don't understand what your question relates to :)

Ok see it now, on the main page.. GH did show your comment on a different diff set after following the email notification and it was totally confusing, sorry.